### ISSUE TITLE
[Security] Windows Defender detects ChatGPT.exe as a Trojan

### ISSUE THREAD

**Issue by acoudray** [lencx/ChatGPT#436]
### Description

Windows Defender detects ChatGPT.exe as a Trojan. Happened today before updating, v0.10.3
![image](https://user-images.githubusercontent.com/24280328/219852683-74211c82-9e04-4227-9472-e33d5f4e0a16.png)

**Comment by DeveloCraft** [comment]
Me too, just now !

**Comment by DioVayne** [comment]
Same here
![image](https://user-images.githubusercontent.com/2700848/219853010-092ed511-0451-4609-85cc-289041139da9.png)

**Comment by LukaErnestini** [comment]
Same for me!

**Comment by Saya47** [comment]
Same lol

**Comment by warshell** [comment]
++++

**Comment by shanedk** [comment]
I got the same thing. I updated to the latest version, and now it's "Malicious.moderate.ml.score" instead of the Malgent trojan.

Microsoft is the only one marking this as malware. https://www.virustotal.com/gui/file/471b3b4bd2c8739330d9a6405bf0289d5dc4a98d5b13b3cd40c667e43bd341e1/detection

**Comment by jonasmarco** [comment]
Same here

**Comment by yixinBC** [comment]
I have reported this issue to Microsoft.
https://aka.ms/AAjp4bx

**Comment by 00xSky** [comment]
I don't think this virus is caused by Microsoft or a 3rd party reason. Considering the feedback sent to you and my use case on 2 different computers, I'm pretty sure this virus originated from you. I do not want to approach this situation from a biased point of view, so I would greatly appreciate it if you could explain the reasons that led me to this thought.

I installed it on 2 different computers 1 week ago, 3 days apart, and after running the "ChatGPT.exe" file for a total of 24 hours, update notifications came to both computers. After the update process was completed, the file was automatically deleted because Microsoft found a virus. 

Now, the update process after 24 hours of use may be a coincidence, but the virus situation takes the matter to a very different point. This suggests that it could be more than just a coincidence. Currently, this is not an issue for someone who downloads and installs the app from GitHub.

In the first installation, everything works smoothly and there is no problem with the virus, why is there a virus warning after 24 hours of use?Also, please let's not limit this topic to just "Microsoft". You can understand more clearly when you scan your update file through "virustotal"

**Comment by DioVayne** [comment]
Now I even get suspicious as to why it's taking such a long time to uninstall.

**Comment by yixinBC** [comment]
> Now I even get suspicious as to why it's taking such a long time to uninstall.

Sometimes uninstall a program from `.msi` installer is slow.I must point out that this is almost the common problem of this type of installation

**Comment by wanzhiyu** [comment]
Me too. I use choco install this app. Windows denfnder shows "Trojan:Win64/Malgent!MSR".

**Comment by oriofdarkforest** [comment]
Windows 11 has also reported this.

**Comment by zhixiangjoy** [comment]
Just now, I meet the same problem.

**Comment by 00xSky** [comment]
Can anyone share the post-update virus files here? I think the only evidence we will get an answer on this issue is the infected files.

**Comment by mliyuanjie** [comment]
I also got this reports, i am sure this is from the softeare. could anyone explain the Trojan reports? this should not happend, if so we need report this situation to github.

**Comment by tribixbite** [comment]
i'm guessing he's right lol
![image](https://user-images.githubusercontent.com/381345/220431082-5625eb05-0a44-4cd0-b2c4-09f626468809.png)

**Comment by ann7bel** [comment]
> I also got this reports, i am sure this is from the softeare. could anyone explain the Trojan reports? this should not happend, if so we need report this situation to github.

Then, what happened with defender? It was a false positive?

**Comment by lencx** [comment]
I've updated the readme. security agencies have now detected a trojan in the unknown download link.

**🛑 URGENT NOTICE: A hacker has been found to take advantage of the heat of `lencx/ChatGPT` to plant a Trojan horse after the fork project and rebuild the installer. If you have friends around you who are using this desktop application, please remind them not to download unknown links freely. Now the project will remove other installation ways and only provide this download link https://github.com/lencx/ChatGPT/releases**

**🛑 紧急通知：目前发现有黑客利用 `lencx/ChatGPT` 的热度，在 fork 项目后植入木马，重新构建安装程序。如果你身边有朋友正在使用此桌面应用，请提醒 TA 们不要随意下载不明链接。现在项目将删除其他安装途径，仅提供此下载链接 https://github.com/lencx/ChatGPT/releases**

**Comment by Ethkuil** [comment]
> Now the project will remove other installation ways and only provide this download link [lencx/ChatGPT/releases](https://github.com/lencx/ChatGPT/releases)

I use `winget` to install this software. Can this ensure safety?

**Comment by lencx** [comment]
> I use `winget` to install this software. Can this ensure safety?

It is safe.

**Comment by oriofdarkforest** [comment]
> 

My client is just downloaded from the release page. Someday Microsoft forced to delete it.
Not the first day, but after I used it for about a week.
It did not convince me.

**Comment by DemGiran** [comment]
> > I use `winget` to install this software. Can this ensure safety?
> 
> It is safe.

Scoop ?

**Comment by JiveyGuy** [comment]
Something about this whole repo screams spyware.

**Comment by mgarbade** [comment]
I remeber a similar repo I was using before and there the same spyware / virus discussions came up. Not that this is just copy / paste from that same old haunted repository
